database/models/features.db.php

```diff
@@ -22,25 +23,25 @@ function __construct(User $user, Feature $feature)
         $this->featureName = $feature->value;
         $this->user = $user;
     }
+}
 
-    /**
-     * Get all feature from a user
-     * @return bool
-     */
-    static function hasFeature($uid, $feature)
-    {
-        return !!em()->createQuery("SELECT count(f) FROM UserFeature f WHERE f.user = :u AND f.featureName = :feature AND f.enabled = :enabled")
-            ->setParameters(["u" => $uid, "feature" => $feature, "enabled" => true])
-            ->getSingleScalarResult();
-    }
```

Comment:
Did you intend to remove this method? If yes, why? It's a nice and convenient way to keep the DQL near the entity and prevent errors when we change something in the entity.

(in general, we should aim at always doing something like this and keep the queries either near the entity or in a repository)

Reply:
Moved it to FeatureService :)

Reply:
Fair enough :) that's good for now, let's think later if we want repositories or not